cmd/runtimecfg/render.go

```diff
@@ -36,6 +36,7 @@ func init() {
 	renderCmd.Flags().IPSlice("cloud-ext-lb-ips", nil, "IP Addresses of Cloud External Load Balancers for OpenShift API")
 	renderCmd.Flags().IPSlice("cloud-int-lb-ips", nil, "IP Addresses of Cloud Internal Load Balancers for OpenShift Internal API")
 	renderCmd.Flags().IPSlice("cloud-ingress-lb-ips", nil, "IP Addresses of Cloud Ingress Load Balancers")
+	renderCmd.Flags().String("platform-type", "", "Optional PlatformType of cloud")
```

Comment:
doc nit: this field is not about a cloud, it's just a platform type so more appropriate description is something like `platformType provides the platform name`

Reply:
Done.